vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -103,15 +107,19 @@ const char *ImageViewManager::GetName() const {
 }
 
 XamlView ImageViewManager::CreateViewCore(int64_t tag) {
-  return ReactImage::Create().as<winrt::Canvas>();
+  auto image = ReactImage::Create().as<winrt::Canvas>();
+  auto border = winrt::Border();
```

Comment:
This introduces an extra XAML element in the tree that may be unnecessary.  Grid and StackPanel both allow rounded corners.  Can you switch the Canvas to one of those instead?  This would save an extra element.

Also - we should consider pushing the rounded corners down to Panel in lifted XAML.  Feel free to discuss/open a feature request on WinUI.

Reply:
On top of the border issue, we also need to solve the problem for background on image, currently image brush is set to the image canvas's background so we need an additional container element to be able to draw background. Having extra border element solves both issues. Agree we should consider rounded corners for panel, we have similar issue for ViewViewManager too. 

---
In reply to: [356349285](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r356349285) [](ancestors = 356349285)